src/Confluent.Kafka/Consumer_KV.cs

```diff
@@ -0,0 +1,393 @@
+
+        /// <include file='include_docs_consumer.xml' path='API/Member[@name="Commit_IEnumerable"]/*' />
+        public CommittedOffsets Commit(IEnumerable<TopicPartitionOffset> offsets)
+            => consumer.Commit(offsets);
+
+        /// <include file='include_docs_consumer.xml' path='API/Member[@name="Dispose"]/*' />
+        public void Dispose()
+        {
+            if (KeyDeserializer != null)
+            {
+                KeyDeserializer.Dispose();
+            }
+
+            if (ValueDeserializer != null)
+            {
+                ValueDeserializer.Dispose();
+            }
+
+            consumer.Dispose();
+        }
+
+        /// <include file='include_docs_consumer.xml' path='API/Member[@name="Seek"]/*' />
+        public void Seek(TopicPartitionOffset tpo)
+            => consumer.Seek(tpo);
+
+        /// <include file='include_docs_consumer.xml' path='API/Member[@name="Pause"]/*' />
+        public List<TopicPartitionError> Pause(IEnumerable<TopicPartition> partitions)
+            => consumer.Pause(partitions);
+
+        /// <include file='include_docs_consumer.xml' path='API/Member[@name="Resume"]/*' />
+        public List<TopicPartitionError> Resume(IEnumerable<TopicPartition> partitions)
+            => consumer.Resume(partitions);
+
+        /// <include file='include_docs_consumer.xml' path='API/Member[@name="Committed_IEnumerable_TimeSpan"]/*' />
+        public List<TopicPartitionOffsetError> Committed(IEnumerable<TopicPartition> partitions, TimeSpan timeout)
+            => consumer.Committed(partitions, timeout);
+
+        /// <include file='include_docs_consumer.xml' path='API/Member[@name="Position_IEnumerable"]/*' />
+        public List<TopicPartitionOffsetError> Position(IEnumerable<TopicPartition> partitions)
+            => consumer.Position(partitions);
+
+        /// <include file='include_docs_client.xml' path='API/Member[@name="Client_Name"]/*' />
+        public string Name
+            => consumer.Name;
+
+        /// <include file='include_docs_consumer.xml' path='API/Member[@name="MemberId"]/*' />
+        public string MemberId
+            => consumer.MemberId;
+
+        /// <include file='include_docs_client.xml' path='API/Member[@name="ListGroups_TimeSpan"]/*' />
+        public List<GroupInfo> ListGroups(TimeSpan timeout)
+            => consumer.ListGroups(timeout);
+
+        /// <include file='include_docs_client.xml' path='API/Member[@name="ListGroup_string_TimeSpan"]/*' />
+        public GroupInfo ListGroup(string group, TimeSpan timeout)
+            => consumer.ListGroup(group, timeout);
+
+        /// <include file='include_docs_client.xml' path='API/Member[@name="ListGroup_string"]/*' />
+        public GroupInfo ListGroup(string group)
+            => consumer.ListGroup(group);
+
+        /// <include file='include_docs_consumer.xml' path='API/Member[@name="GetWatermarkOffsets_TopicPartition"]/*' />
```

Comment:
`GetWatermarkOffsets` is only on consumers (not producers). can't remember if there is a good reason for this or not.